src/lib/features/Card.svelte

```diff
@@ -0,0 +1,28 @@
+<script>
+  import { UserFilled } from '/src/lib/assets/index';
+
+  export let title = 'Default Card Header';
+  export let bgColor = 'cream';
+  export let color = 'black'
+  export let titleColor = 'navy'
+  export let icon = UserFilled;
+</script>
+
+<div class="p-6 card-shadow rounded-xl bg-{bgColor} {$$restProps.class}">
+  <div class="pb-6 flex justify-between">
+    <h5 class="font-bold text-{titleColor} inline">{title}</h5>
+    <svelte:component this={icon} class="w-f48 h-f48"/>
+  </div>
+  <div class="text-{color}">
+    <slot name="content" >
+      This is default content
+    </slot>
+  </div>
```

Comment:
```suggestion
    <slot name="content" >
      This is default content
    </slot>
```
The default color will be black, then if the color needs to be changed, it can come through the restProps.class at the top level as well. Let's keep the title color though.